Platform/macOS/UTMApp.swift

```diff
@@ -32,6 +32,9 @@ struct UTMApp: App {
                     data.showErrorAlert(message: message)
                 }
             }
+            .onReceive(NSNotification.ShowMainWindow) { _ in
+                NSApplication.shared.windows.first?.makeKeyAndOrderFront(nil)
```

Comment:
I don't know if this is good enough. I might play around with it but thinking of capturing a reference in the AppDelegate in order to reference it.

Reply:
Sounds good, thanks!